api/object.go

```diff
@@ -83,14 +83,24 @@ type (
 		Object  *Object          `json:"object,omitempty"`
 	}
 
-	// GetObjectResponse is the response type for the /worker/object endpoint.
+	// GetObjectResponse is the response type for the GET /worker/object endpoint.
 	GetObjectResponse struct {
 		Content      io.ReadCloser      `json:"content"`
 		ContentType  string             `json:"contentType"`
 		LastModified string             `json:"lastModified"`
 		Range        *DownloadRange     `json:"range,omitempty"`
 		Size         int64              `json:"size"`
 		Metadata     ObjectUserMetadata `json:"metadata"`
+		// NOTE: keep HeadObjectResponse in sync with this type
```

Comment:
Considering that these two return types are supposed to be the same by definition we might want to make an exception and embed `HeadObjectResponse` here. Because there is no way the can ever diverge.

Reply:
I actually originally had it embedded, when preparing the PR though I figured it'd be best to split it just because we have the rule. I agree though in this case we can make an exception 👍 it's a special case.